cms/templates/container.html

```diff
@@ -110,10 +111,10 @@ <h3 class="sr">${_("Page Actions")}</h3>
                 % if xblock.category == 'split_test':
                     <div class="bit">
                         <h3 class="title-3">${_("Adding components")}</h3>
-                        <p>${_("Select a component type under {em_start}Add New Component{em_end}. Then select a template.").format(em_start='<strong>', em_end="</strong>") | h}</p>
+                        <p>${_(HTML_ESCAPE("Select a component type under {em_start}Add New Component{em_end}. Then select a template.")).format(em_start=HTML('<strong>'), em_end=HTML("</strong>"))}</p>
```

Comment:
This won't localize properly.  The `_()` function must have a string literal as its only argument.  You can do the formatting and escaping on its result.